packages/internal-input-option/src/InputOption.spec.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { InputOption } from '.';
+
+describe('packages/internal-input-option', () => {
```

Comment:
feels like we should be testing a little more here...

`className` prop appears in the classLIst, `onClick` gets called, `disabled` works, etc.

Reply:
children render